eblocker-icapserver/src/main/java/org/eblocker/server/common/blacklist/DomainBlockingService.java

```diff
@@ -223,23 +223,6 @@ public Decision isDomainBlockedByMalwareAdsTrackersFilters(String domain) {
         }
     }
 
-    public List<CachingFilter.Stats> getCacheStats(boolean includeDomains) {
```

Comment:
The IDE showed me this method as no longer used. Just like the clearCache method. That's where it all started.